shopelectro/models.py

```diff
@@ -121,6 +120,10 @@ def get_siblings(self, offset):
             .select_related('page')[:offset]
         )
 
+    def get_brand_name(self) -> str:
```

Comment:
if we decide to create `get_brand` method, we can rm it.
See get_brand method proposal [here](https://github.com/fidals/shopelectro/pull/531/files#diff-363c06261e0934f61f73b3e7004f01c4R237)